tests/scripts/mbedtls_test.py

```diff
@@ -90,20 +89,24 @@ def __parse(self, data_f):
         :param data_f: Data file object
         :return:
         """
-        for line in data_f:
-            line = line.strip()
+        while True:
+            line = data_f.readline().strip()
             if not line:
-                continue
+                break
             # Read test name
             name = line
 
             # Check dependencies
             dependencies = []
-            line = data_f.next().strip()
+            line = data_f.readline().strip()
```

Comment:
Can we use `line = next(data_f).strip()` in both python2.7 and python3?

Reply:
I haven't tried. I'll check

Reply:
If we can use this, maybe we don't have to change the loop that much.

Reply:
I agree, and I think it will work. I am currently working on the false positive I encountered

Reply:
Looks good